packages/compiler/src/myst.ts

```diff
@@ -0,0 +1,41 @@
+import { micromark } from "micromark";
```

Comment:
Getting a lot of typescript errors due to the lack of types. I don't think fixing them would be *too* hard (just follow the example of the other files) but I'd be willing to accept a PR that just uses a `.js` file for now.

Reply:
Added a panel type to ```types.ts```